stixcore/processing/pipeline_cron.py

```diff
@@ -0,0 +1,436 @@
+import io
+import re
+import sys
+import shutil
+import socket
+import inspect
+import logging
+import smtplib
+import warnings
+import importlib
+import threading
+import subprocess
+from queue import Queue
+from pprint import pformat
+from pathlib import Path
+from datetime import datetime
+from configparser import ConfigParser
+
+from polling2 import poll_decorator
+from watchdog.events import FileSystemEventHandler
+
+import stixcore
+from stixcore.config.config import CONFIG
+from stixcore.ephemeris.manager import Spice, SpiceKernelManager
+from stixcore.idb.manager import IDBManager
+from stixcore.io.RidLutManager import RidLutManager
+from stixcore.io.soc.manager import SOCPacketFile
+from stixcore.processing.L0toL1 import Level1
+from stixcore.processing.LBtoL0 import Level0
+from stixcore.processing.TMTCtoLB import process_tmtc_to_levelbinary
+from stixcore.products import Product
+from stixcore.soop.manager import SOOPManager
+from stixcore.util.logging import STX_LOGGER_DATE_FORMAT, STX_LOGGER_FORMAT, get_logger
+from stixcore.util.singleton import Singleton
+from stixcore.version_conf import get_conf_version
+
+__all__ = ['GFTSFileHandler', 'process_tm', 'PipelineErrorReport', 'PipelineStatus']
+
+logger = get_logger(__name__)
+warnings.filterwarnings('ignore', module='astropy.io.fits.card')
+warnings.filterwarnings('ignore', module='astropy.utils.metadata')
+warnings.filterwarnings('ignore', module='watchdog.events')
+
+TM_REGEX = re.compile(r'.*PktTmRaw.*.xml$')
+
+
+class GFTSFileHandler(FileSystemEventHandler):
```

Comment:
Is this class used here now, maybe a left over from a move/rename?

Reply:
yes will be removed